### ISSUE TITLE
Website syntax highlighting bug

### ISSUE THREAD

**Issue by DreamSaddle** [element-plus/element-plus#363]
Missing indentation and highlighting of component documentation code blocks, it feels more difficult to read. Hope to optimize, thank you!
---
**组件文档代码块缺失缩进与高亮, 感觉比较影响阅读. 希望优化,谢谢!**

![image](https://user-images.githubusercontent.com/31559386/94392775-d0cb5300-018b-11eb-98bc-2263931ea60d.png)

**Comment by valentinpalkovic** [comment]
I would like to give it a try to write issues for element-plus in plain English, that everyone in the world can understand the issue and to avoid duplicate issues. In the original Element-UI library it is nowadays impossible to go through the issues, without understanding mandarin or another Chinese dialect. I think English is the most established language among developers and we should strongly keep this habit. Otherwise, it is a strong argument against element-plus or element-ui to use it in an enterprise business context.

@zazzaz What do you think? Can we write this down into the contribution/issue guidelines?

**Comment by DreamSaddle** [comment]
The problem has been fixed, thanks
